src/index.js

```diff
@@ -0,0 +1,207 @@
+'use strict';
+
+// State tracking object
+const weather = {
+  temperature: 72,
+  clouds: 0,
+  condition: 'Clear',
+  city: 'Atlanta',
+  updating: false,
+};
+
+// Add options to select menu
+const skies = ['☀️', '🌤', '⛅️', '🌥', '🌦', '🌧', '🌈'];
+const skyMenu = document.getElementById('sky-selector');
+
+const createOption = (emoji) => {
+  const opt = document.createElement('option');
+  opt.textContent = emoji;
+  if (emoji === '☀️') {
+    opt.selected = true;
+  }
+  return opt;
+};
+
+for (const skyOption of skies.map(createOption)) {
+  skyMenu.appendChild(skyOption);
+}
+
+// Helper functions used in updates
+const tempClass = () => {
+  if (weather.temperature > 87) {
+    return 'hot';
+  }
+  if (weather.temperature > 77) {
+    return 'warm';
+  }
+  if (weather.temperature > 67) {
+    return 'goldilocks';
+  }
+  if (weather.temperature > 57) {
+    return 'cool';
+  }
+  return 'cold';
+};
+
+const skyClass = () => {
+  if (weather.clouds < 11) {
+    return '☀️';
+  }
+  if (weather.clouds < 25) {
+    return '🌤';
+  }
+  if (weather.clouds < 51) {
+    return '⛅️';
+  }
+  if (
+    weather.clouds < 85 &&
+    weather.condition in { Rain: '', Drizzle: '', Thunderstorm: '' }
+  ) {
+    return '🌦';
+  }
+  if (
+    weather.clouds >= 85 &&
+    weather.condition in { Rain: '', Drizzle: '', Thunderstorm: '' }
+  ) {
+    return '🌧';
+  }
+  return '🌥';
+};
+
+// Manual triggering of change event on select menu
+const changeSkySelection = () => {
+  const selectElement = document.querySelector('#sky-selector');
+  selectElement.value = `${skyClass()}`;
+  selectElement.dispatchEvent(new Event('change'));
+};
+
+// Update temperature display and landscape
+const updateTempDisplay = () => {
+  document.querySelector('#temp-display h1').textContent = weather.temperature;
+  document.getElementById('temp-display').classList = `${tempClass()}`;
+  document.getElementById('landscape').classList = tempClass(
+    weather.temperature
+  );
+};
+
+// Handles clicks on temp controls
+const changeTemp = (e) => {
+  console.log(e.target.id);
+  e.target.id === 'heat' ? weather.temperature++ : weather.temperature--;
+  updateTempDisplay();
```

Comment:
Nice, I like that you have one method to handle both the increase and decrease functionality